### PR TITLE
add example for gaudi2.

### PR DESCRIPTION
**Pull request by lkk12014402**
the differences are:
1. add hpu device
2. load model with torch.bfloat16
3. import AutoRound after loading model
 
**note:** need --amp

### CODE REVIEW

File: examples/language-modeling/main.py [hunk 1]
```diff
@@ -150,24 +147,31 @@ def get_library_version(library_name):
     if model_name[-1] == "/":
         model_name = model_name[:-1]
     print(model_name, flush=True)
-
+    torch_dtype = "auto"
     if args.device == "cpu":
         device_str = "cpu"
+    elif args.device == "cuda":
```
Comment (wenhuach21) [inline examples/language-modeling/main.py]: cuda in device

Comment (wenhuach21) [reply examples/language-modeling/main.py]: there are some cases users pass "cuda:4"

File: examples/language-modeling/run_autoround_on_gaudi.sh [hunk 2]
```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -x
+model_name="Intel/neural-chat-7b-v3-3"
+eval_bs6=1
```
Comment (wenhuach21) [inline examples/language-modeling/run_autoround_on_gaudi.sh]: typo

File: examples/language-modeling/main.py (hunk 1 above)
Comment (lkk12014402) [reply examples/language-modeling/main.py]: fixed

File: examples/language-modeling/run_autoround_on_gaudi.sh (hunk 2 above)
Comment (lkk12014402) [reply examples/language-modeling/run_autoround_on_gaudi.sh]: fixed

**Pull request merged**